packages/augur-ui/src/modules/account/components/account-view.styles.less

```diff
@@ -170,23 +191,50 @@
       }
     }
 
+    > div:first-of-type {
+      order: 1;
+    }
+
     > div:nth-of-type(2) {
-      > div[class*='Headers'] {
-        background: var(--color-table-header-50);
-        border-top-left-radius: @border-radius-rounded;
-        border-top-right-radius: @border-radius-rounded;
+      order: 2;
+      min-height: @size-744;
+      max-height: @size-744;
+    }
 
-        > ul > li {
-          border-bottom: none;
+    > div:nth-of-type(3) {
+      order: 2;
+      min-height: @size-550;
+      max-height: @size-550;
+    }
 
-          > button {
-            .text-16-bold;
+    > div:nth-of-type(4) {
+      order: 3;
+      min-height: @size-160;
+      max-height: @size-160;
+      margin-right: 0;
+    }
 
-            text-transform: capitalize;
-            height: @size-56;
-          }
-        }
-      }
+    > div:nth-of-type(5) {
+      order: 3;
+      margin-right: 0;
+    }
+
+    > div:nth-of-type(6) {
+      order: 1;
+    }
+
+    > footer {
+      min-height: @size-40;
     }
   }
+
+  /* Force new columns */
+  .AccountView::before,
+  .AccountView::after {
```

Comment:
I would rather use grid to avoid things like this

Reply:
Removed.